:art: fixed inconsinstent example code indentations

jannes-io opened the pull request:
Some of the code examples used 2 spaces, and some were wrongly indented. This PR fixes the inconsistent JavaScript indentation to 4 spaces, and ensures that opening and closing brackets/braces are placed on the same indentation

jannes-io committed:
:art: fixed inconsinstent example code indentations

Some of the code examples used 2 spaces, and some were wrongly indented. This PR fixes the inconsistent JavaScript indentation to 4 spaces, and ensures that opening and closing brackets/braces are placed on the same indentation

stianst commented:
@jannes-io thanks for your PR. I'm afraid there was another PR earlier in the queue that changes this file as well so now there are some conflicts. Would you be able to rebase/resolve conflicts?

jannes-io committed:
Merge branch 'master' into patch-1

jannes-io commented:
@stianst no problem. I fixed the merge conflicts and updated according to master. Also brought those blocks into the same codestyle as the examples in the upper half of the doc.

stianst commented:
@jannes-io awesome, thanks for the quick turnaround.
